### PR TITLE
Add new tile and wall

### PR DESCRIPTION
**Pull request by STEdirisuriya**
This pull request is related to issue #https://github.com/fossasia/labyrinth/issues/44

I added tile and wall, you can test my changes here:
http://rawgit.com/STEdirisuriya/labyrinth/master/index.html

### CODE REVIEW

File: js/tiles.js [hunk 1]
```diff
@@ -293,3 +293,11 @@ const door = {
...
   }),
 };
+outdoor: Object.assign({}, OpenDoors, {
+   	canEnterFromTheRight() {return false;},
+      canLeaveToTheRight() {return false;},
+      createImages: function() {
+          this.wallRight = this.createImage("tiles/rooms/wall/outwall.svg");
+          this.ground = this.createImage("tiles/rooms/floor/outdoor.svg");
+      },
+}),
```
Comment (naveentherocker) [inline js/tiles.js]: Remove the `,`

File: js/tiles.js [hunk 2]
```diff
@@ -293,3 +293,11 @@ const door = {
     },
   }),
 };
+outdoor: Object.assign({}, OpenDoors, {
```
Comment (abishekvashok) [inline js/tiles.js]: Please properly add this object to the door class.

File: js/tiles.js (hunk 1 above)
Comment (abishekvashok) [reply js/tiles.js]: It's okay to leave that there. But the position is wrong :)

File: js/tiles.js (hunk 2 above)
Comment (niccokunzmann) [reply js/tiles.js]: This needs to be one line higher.

     };

must be at the end of the file.

Comment (niccokunzmann) [reply js/tiles.js]: I guess, this is what @abishekvashok wants to say.

**Comment by niccokunzmann** [comment]
If you open the console in the browser, you should be able to see and error. In Firefox do: right click + Inspect Element. Then, click on console.

File: js/tiles.js [hunk 3]
```diff
@@ -292,4 +292,12 @@ const door = {
       this.ground.show();
     },
   }),
+outdoor: Object.assign({}, OpenDoors, {
```
Comment (abishekvashok) [inline js/tiles.js]: Can you properly intend this pls?

**Comment by STEdirisuriya** [comment]
I can't see my changes from that link. When I go to the link nothing there and after there is a button How to Play. After I click it I went to fossasia lybrinth game.

**Comment by abishekvashok** [comment]
You need to add it to `levels.js` as well.

**Comment by STEdirisuriya** [comment]
@abishekvashok how to add it into levels.js

**Comment by niccokunzmann** [comment]
Please see the link: http://rawgit.com/STEdirisuriya/labyrinth/master/index.html and the two errors in the console. What would you like to do?

**Comment by abishekvashok** [comment]
Note: Solving the 1st one will solve the second.

**Comment by STEdirisuriya** [comment]
is it ok now

**Comment by niccokunzmann** [comment]
I cannot play the game, can you? http://rawgit.com/STEdirisuriya/labyrinth/master/index.html

**Comment by abishekvashok** [comment]
Same error. See the gitter channel.

File: js/tiles.js [hunk 4]
```diff
@@ -292,4 +292,12 @@ const door = {
...
     },
   }),
+  outdoor: Object.assign({}, OpenDoors, {
+    canEnterFromTheRight() { return false; },
+    canLeaveToTheRight() { return false; },
+    createImages: function () {
+      this.wallRight = this.createImage("tiles/rooms/wall/outwall.svg");
+      this.ground = this.createImage("tiles/rooms/floor/outdoor.svg");
+    },
+  });
```
Comment (niccokunzmann) [inline js/tiles.js]: I think, the ; is too much. It could be a , instead.

Comment (niccokunzmann) [reply js/tiles.js]: I will merge it as soon as I can play it.

Comment (STEdirisuriya) [reply js/tiles.js]: oh yes

**Comment by STEdirisuriya** [comment]
@niccokunzmann Check it now!

**Comment by niccokunzmann** [comment]
Wow! Nice tile, finally I see it. 
Now, I would like to merge it. But we have a conflict because someone edited the files in the same location.
Would it be ok for you to resolve the conflict?

**Comment by abishekvashok** [comment]
Please resolve the conflicts @STEdirisuriya

**Comment by Ploypiti** [comment]
@STEdirisuriya Squash your commits as well.

**Comment by niccokunzmann** [comment]
Thanks for your contribution and taking the time to complete it!

**Comment by STEdirisuriya** [comment]
@niccokunzmann Sure! thanks

**Pull request merged**